test/helpers/kubectl.go

```diff
@@ -1396,6 +1396,14 @@ func (kub *Kubectl) ciliumInstallHelm(filename string, options map[string]string
 		return err
 	}
 
+	// Remove cilium DS to ensure that new instances of cilium-agent are started
+	// for the newly generated ConfigMap. Otherwise, the CM changes will stay
+	// inactive until each cilium-agent has been restarted.
+	_ = kub.DeleteResource("ds", fmt.Sprintf("-n %s cilium", GetCiliumNamespace(GetCurrentIntegration())))
```

Comment:
Why can't we just use `deleteCiliumDS`? Also, you should be able to use `CiliumNamespace` directly.

Reply:
I've removed that function. Previously, it was defined in `test/k8sT/assertionHelpers.go`. So, using it from that module would introduce a circular dependency.